partials/post-card.hbs

```diff
@@ -15,7 +15,7 @@
                     </span>
                 {{/primary_tag}}
             {{else}} 
-                <a class="text-lg uppercase font-extrabold">[No tag]</a>
+                <b class="text-lg uppercase font-extrabold">[No tag]</b>
```

Comment:
Could we use the `p` tag instead to make it clear that the font weight is being styled by Tailwind, since the utility class `font-extrabold` is applied here?